.coveragerc

```diff
@@ -974,6 +974,7 @@ omit =
     homeassistant/components/shelly/climate.py
     homeassistant/components/shelly/entity.py
     homeassistant/components/shelly/light.py
+    homeassistant/components/shelly/number.py
```

Comment:
```suggestion
```
I don't think we should continue to add platforms without tests. Besides the `climate` platform which merged without tests all our last added platforms - `device_trigger`, `logbook`, button` added with tests.

Reply:
I don't think this is a issue, we can always add them later. Isn't it ?

Simone